client/views/blocks/ModalBlock.js

```diff
@@ -59,6 +59,12 @@ function ModalBlock({ view, errors, appId, onSubmit, onClose, onCancel }) {
 		(event) => {
 			if (event.keyCode === 13) {
 				// ENTER
+				if (
+					event.srcElement &&
+					(event.srcElement.nodeName === 'TEXTAREA' || event.srcElement.localName === 'textarea')
+				) {
+					return;
+				}
 				return onSubmit(event);
```

Comment:
In my point of view, is not necessary return something, how about if/else block only?

Reply:
Yeah makes sense. I've modified it in a way to avoid blank return